sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -290,6 +291,29 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       checkAnswer(
         sql("SELECT employeeID, employeeName FROM part_table WHERE c = '2' AND d = '1'"),
         sql("SELECT * FROM non_part_table").collect())
+
+      // check for case insensitive property of partition column name in load command.
+      withSQLConf(SQLConf.CASE_SENSITIVE.key -> "false") {
+        withInputFile { f =>
+          sql(s"""$loadQuery INPATH "${f.toURI}" INTO TABLE part_table PARTITION(C="1", D="2")""")
+        }
+        checkAnswer(
+          sql("SELECT employeeID, employeeName FROM part_table WHERE c = '2' AND d = '1'"),
+          sql("SELECT * FROM non_part_table").collect())
+      }
+    }
+  }
+
+  test("case insensitive property of partition column name in insert command") {
```

Comment:
Since this is a bug fix, we need JIRA ID for the test case name.
```
- test("case insensitive property of partition column name in insert command") {
+ test("SPARK-28084 case insensitive property of partition column name in insert command") {
```

Reply:
done